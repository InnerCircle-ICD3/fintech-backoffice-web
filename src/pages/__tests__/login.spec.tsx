import { expect, test } from 'vitest';
import Login from '@/pages/auth/login';
import { render } from '@testing-library/react';

test('Login', () => {
  const { container } = render(<Login />);
  expect(container).toMatchSnapshot();
});
