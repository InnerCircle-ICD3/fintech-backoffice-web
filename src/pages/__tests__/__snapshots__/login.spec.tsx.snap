// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Login 1`] = `
<div>
  <div>
    로그인 페이지 Index
  </div>
</div>
`;

exports[`로그인 컴포넌트의 스냅샷이 일치해야 함 1`] = `
<div>
  <div>
    로그인 페이지 Index
  </div>
</div>
`;
